pkg/storage/node_liveness.go

```diff
@@ -48,6 +48,7 @@ var (
 
 // Node liveness metrics counter names.
 var (
+	metaLiveNodes          = metric.Metadata{Name: "liveness.livenodes"}
```

Comment:
can you add `Help` fields to each of those metrics describing what they are. See `/_status/vars` for various examples.